src/js/base/ffi-helpers.js

```diff
@@ -456,6 +456,8 @@ define(["js/runtime-util", "trove/lists", "trove/sets", "trove/option", "trove/e
 
         makeList: makeList,
         makeTreeSet: makeTreeSet,
+
+        isOption: runtime.getField(O, "is-Option"),
```

Comment:
@blerner @belph I know the reason why it shouldn't call `.app` now: so that it can be used in `ffi.cases`, because `ffi.cases` will automatically call `.app` for us.